torch/fx/_graph_pickler.py

```diff
@@ -565,3 +565,13 @@ def unpickle(self, unpickle_state: _UnpickleState) -> TracingContext:
             self.force_unspec_int_unbacked_size_like
         )
         return context
+
+
+class SubprocGraphPicker(torch._inductor.compile_worker.subproc_pool.SubprocPickler):
+    @override
```

Comment:
comment about what this is?

Reply:
Another bit of code that belongs in a future PR.  Moved.